UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Microsoft.Extensions.Logging;
+
+namespace UKHO.PeriodicOutputService.Common.Logging
+{
+    public enum EventIds
+    {
+        /// <summary>
+        /// 809000 - Periodic Output Service webjob request started 
+        /// </summary
```

Comment:
Please correct syntax wherever required.
```suggestion
        /// </summary>
```

Reply:
Corrected